src/main/java/com/minecraftabnormals/abnormals_core/core/AbnormalsCore.java

```diff
@@ -152,7 +153,10 @@ private void clientSetup(FMLClientSetupEvent event) {
 	}
 
 	private void postLoadingSetup(FMLLoadCompleteEvent event) {
-		event.enqueueWork(() -> DataUtil.getSortedAlternativeDispenseBehaviors().forEach(DataUtil.AlternativeDispenseBehavior::register));
+		event.enqueueWork(() -> {
+			DataUtil.getSortedAlternativeDispenseBehaviors().forEach(DataUtil.AlternativeDispenseBehavior::register);
+			ACEvents.SORTED_CUSTOM_NOTE_BLOCK_INSTRUMENTS.addAll(DataUtil.getSortedCustomNoteBlockInstruments());
```

Comment:
Why not just set `ACEvents.SORTED_CUSTOM_NOTE_BLOCK_INSTRUMENTS` to `DataUtil.getSortedCustomNoteBlockInstruments()`?

Reply:
Done.